soco/services.py

```diff
@@ -336,14 +432,15 @@ def build_command(self, action, args=None):
         # is set over the network
         return (headers, body)
 
-    def send_command(self, action, args=None, cache=None, cache_timeout=None):
+    def send_command(self, action, args=None, cache=None, cache_timeout=None,
+                     **kwargs):
```

Comment:
Well let's hope that none of the actions have a `cache` argument, or we will be in trouble ;)

Reply:
The argument names are all capitalized, so no need to worry :)

Reply:
:)